src/PatientChecking/PatientChecking/Views/Patient/Index.cshtml

```diff
@@ -1,5 +1,163 @@
-﻿@{
+﻿@model PatientChecking.Views.ViewModels.PatientListViewModel
+@{
     ViewData["Title"] = "Patient";
     ViewData["Page"] = "List Patient";
+    int sortOption = Model.SortSelection;
+    int pageSize = Model.PageSize;
+    int pageIndex = Model.PageIndex;
+    int totalCount = Model.TotalCount;
 }
-<p>This is list patient view</p>
+@section css {
+    <link href="@Url.Content("~/css/PatientList.css")" rel="stylesheet" type="text/css" />
+}
+<div class="search-container">
+    <div class="search-area">
+        <input class="form-control"
+               type="text"
+               placeholder="Search"
+               aria-label="Search" />
+    </div>
+    <div class="sort-area">
+        <select id="sort-selection" class="form-select" style="line-height: 1.5">
+            <option value="0">Sort By ID</option>
+            <option value="1">Sort By Name</option>
+            <option value="2">Sort By DoB</option>
+        </select>
+    </div>
+</div>
+<div class="list-patient-container">
+    <div class="grid wide">
+        @foreach (var item in Model.Patients)
+        {
+            <div class="row">
+                <div class="col l-4">
+                    <div class="item patient-item1">
+                        <div class="patient-img-box">
+                            <img src="~/Image/no-avatar.jpg" alt="Avatar" style="height: 135px" />
+                        </div>
+                        <div class="patient-name">
+                            <p style="font-weight: 600; font-size: 18px">@item.FullName</p>
```

Comment:
Should move this  style to a css class, please don't use inline style. Please check for other places.

Reply:
Inline style has been all removed in code. Please check this for me again.